README.md

```diff
@@ -653,6 +653,51 @@ Parameter | Description
 --pyspark | Generates the Spark SQL file.
 --scala | Generates the Scala SQL file.
 
+#### query
```

Comment:
The query command allows you to use natural language commands to query your databases, transforming these commands into SQL SELECT statements. By leveraging the capabilities of AI and LLMs, specifically OpenAI models, it interprets user queries and generates the corresponding SQL queries. For effective use of this command, users need to provide their OpenAI API Key and specify the OpenAI model to be utilized. The output will be written to a CSV file. The max number of rows that will be returned is 200. You can overwrite this value, or remove completely the limit.

Reply:
Here as a last sentence we can add that:

**The default openai model that is used is gpt-3.5-turbo**